woocommerce/woocommerce-android
# Hub Menu: Add Avatar and Site Name

Closes: #5584


### Description


This shows user avatar, site name, and site URL on top left of the site.

<img src="https://user-images.githubusercontent.com/266376/151752501-f1648a29-d207-4db1-806b-bc181207b095.png" width="42%" />


### Testing instructions
Run app and go to Hub menu. Make sure user avatar, site name, and site URL are shown properly.
